python/backend/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# https://eng.ms/docs/more/containers-secure-supply-chain/approved-images
-FROM mcr.microsoft.com/oryx/python:3.9
+FROM python:3.9-slim AS builder
```

Comment:
won't our devops complain about this?

Reply:
You are right, I think we should follow this guidance here: https://eng.ms/docs/more/containers-secure-supply-chain/approved-images#how-to-handle-externally-visible-dockerfiles